packages/Python/lldbsuite/test/lang/swift/hashed_containers_enums/TestSwiftHashedContainerEnum.py

```diff
@@ -0,0 +1,97 @@
+# TestSwiftHashedContainerEnum.py
+
+"""
+Test combinations of hashed swift containers with enums as keys/values
+"""
+
+import lldb
+from lldbsuite.test.lldbtest import *
+import lldbsuite.test.decorators as decorators
+import lldbsuite.test.lldbutil as lldbutil
+import os
+import unittest2
+
+class TestSwiftHashedContainerEnum(TestBase):
+
+    mydir = TestBase.compute_mydir(__file__)
+
+    @decorators.skipUnlessDarwin
+    @decorators.swiftTest
+    def test_any_object_type(self):
+        """Test combinations of hashed swift containers with enums"""
+        self.build()
+        self.do_test()
+
+    def setUp(self):
+        TestBase.setUp(self)
+        self.main_source = "main.swift"
+        self.main_source_spec = lldb.SBFileSpec(self.main_source)
+
+    def do_test(self):
+        """Test combinations of hashed swift containers with enums"""
+        exe_name = "a.out"
+        exe = os.path.join(os.getcwd(), exe_name)
+
+        # Create the target
+        target = self.dbg.CreateTarget(exe)
+        self.assertTrue(target, VALID_TARGET)
+
+        # Set the breakpoints
+        breakpoint = target.BreakpointCreateBySourceRegex(
+            '// break here', self.main_source_spec)
+        self.assertTrue(breakpoint.GetNumLocations() > 0, VALID_BREAKPOINT)
+
+        # Launch the process, and do not stop at the entry point.
+        process = target.LaunchSimple(None, None, os.getcwd())
+
+        self.assertTrue(process, PROCESS_IS_VALID)
+
+        # Frame #0 should be at our breakpoint.
+        threads = lldbutil.get_threads_stopped_at_breakpoint(
+            process, breakpoint)
+
+        self.assertTrue(len(threads) == 1)
+        self.thread = threads[0]
+        self.frame = self.thread.frames[0]
+        self.assertTrue(self.frame, "Frame 0 is valid.")
+
+        # TODO: This should work after https://github.com/apple/swift-lldb/pull/259
```

Comment:
In theory, you could base branch on that branch, but does swift-ci handle that structure?

Reply:
Yea, until I saw the green this morning on #259 I wasn't sure when that would be merged. I will rebase once I get an answer on the linux tests over there

Reply:
> does swift-ci handle that structure?

Based on [this document](https://github.com/apple/swift/blob/9a5df02e5e0bae3ffff9b1f0590e92d46366ee9f/docs/ContinuousIntegration.md#cross-repository-testing), I think it might. I'll try.